.github/workflows/cd-deploy.yml

```
@@ -35,20 +30,21 @@ jobs:
     outputs:
-      frontend-bucket:   ${{ steps.pulumi.outputs.FrontendBucketName }}
-      artifacts-bucket:  ${{ steps.pulumi.outputs.S3ArtifactsBucket }}
-      eb-app-name:       ${{ steps.pulumi.outputs.EbAppName }}
-      eb-env-name:       ${{ steps.pulumi.outputs.EbEnvName }}
-      ecr-url:           ${{ steps.pulumi.outputs.EcrRepositoryUrl }}
+      frontend-bucket:      ${{ steps.pulumi.outputs.FrontendBucketName }}
+      artifacts-bucket:     ${{ steps.pulumi.outputs.S3ArtifactsBucket }}
+      eb-app-name:          ${{ steps.pulumi.outputs.EbAppName }}
+      eb-env-name:          ${{ steps.pulumi.outputs.EbEnvName }}
+      ecr-url:              ${{ steps.pulumi.outputs.EcrRepositoryUrl }}
+      lambda-function-name: ${{ steps.pulumi.outputs.LambdaFunctionName }}
```

actionlint (1.7.12)

[error] property "lambdafunctionname" is not defined in object type {output: string}

(expression)